.github/workflows/test-e2e-fork.yml

```diff
@@ -28,14 +28,13 @@ jobs:
       OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
 
   update-check-status:
-    needs: e2e-tests
+    needs: run-e2e-tests-fork
     runs-on: ubuntu-latest
     if: always() && github.event_name == 'repository_dispatch'
     steps:
       - uses: actions/github-script@v6
         env:
           ref: ${{ github.event.client_payload.pull_request.head.sha }}
-          conclusion: ${{ needs.e2e-tests.result }}
```

Comment:
Why is this removed? We need this to set the update job's result for the e2e test.

Reply:
Put it back to use to update e2e status check